src/Raven.Client/Raven.Client.csproj

```diff
@@ -23,6 +23,10 @@
     <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
     <Product>RavenDB Client</Product>
     <Title>RavenDB Client</Title>
+    <Configurations>Debug;Release;Validate</Configurations>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Validate|netstandard1.3|AnyCPU'">
```

Comment:
This will probably not be applied for netstandard2.0

Reply:
Good catch. I  did this when we were already on netstandard2.0, wondering why it didnt applied it at the same time.

Reply:
Probably removing '$(TargetFramework)' from condition will work for you.

Reply:
Note that Sparrow and Client are compiling for netstandard1.3 and 2.0 now.